Client/core/Graphics/CRenderItemManager.cpp

```diff
@@ -328,8 +327,7 @@ void CRenderItemManager::NotifyContructRenderItem(CRenderItem* pItem)
 ////////////////////////////////////////////////////////////////
 void CRenderItemManager::NotifyDestructRenderItem(CRenderItem* pItem)
 {
-    assert(MapContains(m_CreatedItemList, pItem));
-    MapRemove(m_CreatedItemList, pItem);
+    assert(!m_CreatedItemList.insert(pItem).second); // assert if item was already in the collection.
```

Comment:
Should be remove

Reply:
Nice to see you.
Do you mean the assert?

Reply:
Fixed.